content/support-matrix/delfin/_index.md

```diff
@@ -17,10 +17,15 @@ Delfin has standardized common API services.
 | Dell EMC                                            | VNX (Block)               | 4.30 or later                      | CLI                     | Block   | √                        | √            | √   | √          | √    | √    | NA    | NA    | NA          | NA    | √                    | √         | ×           | ×               | ×                   | 1\. Before connecting a Dell EMC VNX Block storage device, install Navisphere Secure CLI on delfin installed node.                                                                                      |   |
```

Comment:
According to the code implementation, I find that the statistics of the vmax and unity devices are slightly different. The red part indicates the result after the change. Please check it.
![image](https://user-images.githubusercontent.com/59107077/157819559-dceffff4-29b8-426c-b7ef-0b45813413fd.png)

Reply:
Done. Thank you